src/com/executors/ExecutorServiceExample.java

```diff
@@ -0,0 +1,24 @@
+package com.executors;
+
+import java.util.concurrent.*;
```

Comment:
Don't use a wild card in import. I have already written about it in chat. 
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad